fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -558,27 +563,55 @@ private void increaseLimit() {
         // Note: This runs all of the updates in serial in order to not invoke a race condition
```

Comment:
Is this comment still true?

Reply:
This appears to still be true, at least the way I'm reading the code

Reply:
If mean "If everything were thread safe, a larger pipeline size would be possible" part. Will there be problem to have a non-1 pipeline size at the write part?

Reply:
I removed the hypothetical part of the comment.

Reply:
I think we still want to have that hypothetical comment. At the moment, we can't do anything in parallel, but at some point we will... It probably doesn't matter too much, and I think the main upshot from this change is that we'd need to think about how to pipeline the `.whileTrue`, etc.

Reply:
NP. I'll put it back.